src/components/Typography/ActionSpan.jsx

```diff
@@ -0,0 +1,27 @@
+// @ts-check
+import React from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+
+import styles from './Typography.module.css'
+
+export const ActionSpan = ({ text, ...props }) => (
+  <span
+    id={props.id}
+    className={cn(styles.action_span, props.className)}
+    onClick={props.onClick}
+    onMouseEnter={props.onMouseEnter}
+    onMouseLeave={props.onMouseLeave}
+  >
+    {text}
+  </span>
+)
+
+ActionSpan.propTypes = {
+  text: PropTypes.string.isRequired,
+  id: PropTypes.string,
```

Comment:
We probably want to make `id` required:

https://github.com/peterjmartinson/huygens-site/pull/19/files#r1451918558